using_images/db_images/mongodb.adoc

```diff
@@ -162,15 +162,19 @@ MongoDB settings can be configured with the following environment variables:
 
 |`*MONGODB_NOPREALLOC*`
 |Disable data file preallocation.
-|true
+|True
```

Comment:
i don't know, we stuff the value in the mongo config here:
https://github.com/openshift/mongodb/blob/master/2.4/root/usr/share/container-scripts/mongodb/mongodb.conf.template

so it's up to mongo.  i'd guess mongo is not case sensitive.